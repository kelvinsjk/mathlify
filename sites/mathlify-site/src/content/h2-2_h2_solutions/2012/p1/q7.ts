import { Answer } from '$lib/classes/answer';
import { QED } from '$lib/typesetting/utils';
import { mathlifierDj as mathlifier } from 'mathlifier';
import {
	generateAns,
	generateFracFnDefinition
} from '$content/h2-1_h2_learn/01_functions/04_more/01_self-inverse/02_practice';
import { expressionToPolynomial } from 'mathlify';
import svg from '$static/images/h2/fns/2012p1q7b.svg?raw';

export const answer = new Answer();

// f(x) = ax/(bx-a)
// g(x) = 1/x

const fState: Parameters<typeof generateFracFnDefinition>[0] = {
	// (bx+k)/(cx+a)
	fnType: 'frac',
	b: 1,
	c: 1,
	k: 'k',
	n: 2,
	show: true
};
const fExp = generateFracFnDefinition(fState, { for: true })[1];
// a
{
	const { soln } = generateAns(fState, fExp, 'g', { hideF2: true, qed: true });
	answer.addPart(mathlifier`[Show](#soln-1).`, soln);
}
// b
{
	const soln = `\`\`\` =html\n${svg}\n\`\`\`\n\n`;
	const ans = mathlifier`[Sketch](#soln-2).`;
	answer.addPart(ans, soln);
}
// c
{
	const [num, den] = fExp._getQuotientTerms();
	const { quotient, remainder, result } = expressionToPolynomial(num).longDivide(
		expressionToPolynomial(den)
	);
	const ans = mathlifier`Line of symmetry: ${{}} {y=x.}

Translate the curve ${{}} {y = \\frac{1}{x}}
by ${1}
unit in the positive ${{}} x\\text{-axis}
direction.
\\
Scale the resulting curve by a factor of ${remainder}
parallel to the ${{}} y\\text{-axis}.
\\
Translate the resulting curve by ${quotient}
unit in the positive ${{}} y\\text{-axis}
direction.`;
	const soln = mathlifier`Line of symmetry: ${{}} y = x \\; ${QED}

$${fExp} = ${result}

- Translate the curve ${{}} {y = \\frac{1}{x}}
by ${1}
unit in the positive ${{}} x\\text{-axis}
direction
- Scale the resulting curve by factor ${remainder}
parallel to the ${{}} y\\text{-axis}
- Translate the resulting curve by ${quotient}
unit in the positive ${{}} y\\text{-axis}
direction`;
	answer.addPart(ans, soln);
}
