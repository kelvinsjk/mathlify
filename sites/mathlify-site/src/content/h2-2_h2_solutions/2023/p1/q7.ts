import { Answer } from '$lib/classes/answer';
import { mathlifierDj as mathlifier } from 'mathlifier';
import { Polynomial } from 'mathlify';
import { absTerm } from 'mathlify/fns';
import { generateRange } from '$content/h2-1_h2_learn/01_functions/01_concepts/02_domain-and-range/02_practice-1';
import { absoluteRationalInverse } from '$content/h2-1_h2_learn/01_functions/02_inverse/03_formula/02_practice';
import { domainRestriction } from '$content/h2-1_h2_learn/01_functions/02_inverse/04_restriction/02_practice';
import { compositeExists } from '$content/h2-1_h2_learn/01_functions/03_composite/01_existence/02_practice';
import { QED } from '$lib/utils/typesetting/utils';
import svg from '$static/images/h2/fns/2023p1q7a.svg?raw';

export const answer = new Answer();

// ! Complete

// | (2x+4) / (3-x) |
// equivalent to | (bx+c) / (x+a) |
// from the learn/practice section

const [a, b, c] = [-3, 2, 4];
const num = new Polynomial([b, c]);
const den = new Polynomial([3, -1], { ascending: true });
const exp = absTerm([num, '/', den]);
const state: Parameters<typeof absoluteRationalInverse>[0] &
	Parameters<typeof domainRestriction>[0] = {
	fnType: 'abs',
	a,
	b,
	c,
	restriction: false,
	type: 'left',
	inclusive: true,
	zeroAns: true,
	zeroLeft: true,
	unknownConstants: false,
	definition: false
};
// from part (d) onwards
const restriction = { type: 'left', x: -2, inclusive: true } as const;

// a
{
	const soln = `\`\`\` =html\n${svg}\n\`\`\``;
	const ans = mathlifier`[Sketch](#soln-1).`;
	answer.addPart(ans, soln);
}
// b
const R_f = generateRange({ ...state, restriction: false }, exp).join(' \\cup ');
{
	const soln = mathlifier`$${{}} R_f = ${R_f} \\;${QED}`;
	const ans = mathlifier`${{}}R_f = ${R_f}.`;
	answer.addPart(ans, soln);
}
// c
{
	const { ans, soln } = compositeExists({ f: state, g: state, fg: true }, [exp, exp], {
		gName: 'f'
	});
	answer.addPart(ans, soln);
}
// d
{
	const { ans, soln } = domainRestriction(state, 'greatest');
	answer.addPart(ans, soln);
}
state.restriction = restriction;
// e
{
	const { ans, soln } = absoluteRationalInverse(state, [num, den], { swap: true });
	answer.addPart(ans, soln);
}
