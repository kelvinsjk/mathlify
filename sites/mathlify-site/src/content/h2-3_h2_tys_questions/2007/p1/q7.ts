/* The question here is the copyright of 
- Singapore Examinations and Assessment Board
- Cambridge Assessment International Education
- Ministry of Education, Singapore
- Government of Singapore

We provide this question on Mathlify under fair use for archival purposes.
These are questions more than 10 years old and are difficult to obtain
commercially.

For the newer questions, please purchase a copy of the
"ten year series" from authorized publishers and distributors.
*/

import { e, i, pi } from 'mathlify';
import { Question } from '$lib/classes/question';
import { mathlifierDj as mathlifier } from 'mathlifier';

export const question = new Question();

question.addBody(mathlifier`The polynomial ${'P(z)'}
has real coefficients. The equation ${{}} {P(z)=0}
has a root ${{}} r${e}^{${i}\\theta},
where ${{}} {r>0}
and ${{}} {0 < \\theta < ${pi}.}`);

// a
{
	question.addPart(
		mathlifier`Write down a second root in terms of ${'r'}
and ${'theta'},
and hence show that a quadratic factor of ${'P(z)'}
is $${{}} z^2 - 2rz \\cos \\theta + r^2.`,
		{ marks: 3 }
	);
}
// b
question.addPart(mathlifier`Out of syllabus (roots of unity).`, { marks: 4 });
// c
question.addPart(mathlifier`Out of syllabus (requires previous part).`, { marks: 3 });
