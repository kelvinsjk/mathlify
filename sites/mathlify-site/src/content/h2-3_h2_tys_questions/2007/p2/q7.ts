/* The question here is the copyright of 
- Singapore Examinations and Assessment Board
- Cambridge Assessment International Education
- Ministry of Education, Singapore
- Government of Singapore

We provide this question on Mathlify under fair use for archival purposes.
These are questions more than 10 years old and are difficult to obtain
commercially.

For the newer questions, please purchase a copy of the
"ten year series" from authorized publishers and distributors.
*/

import { Question } from '$lib/classes/question';
import { mathlifierDj as mathlifier } from 'mathlifier';

const sumX = 4626;
const sumX2 = 147691;
const mu = 30;
const alpha = 5;
const n = 150;

export const question = new Question();

question.addBody(mathlifier`A large number of students in a
college have completed a geography project. The time, ${'x'}
hours, taken by a student to complete the project is noted
for a random sample of ${n}
students. The results are summarised by

$${{}} \\sum x = ${sumX}, \\quad \\sum x^2 = ${sumX2.toLocaleString()}.`);

// a
question.addPart(mathlifier`Find unbiased estimates of the population mean and variance.`, {
	marks: 2
});
// b
question.addPart(
	mathlifier`Test, at the ${alpha}\\%
significance level, whether the population mean time for a student to complete
the project exceeds ${mu}
hours.`,
	{ marks: 4 }
);
// c
question.addPart(
	mathlifier`State, giving a reason, whether any assumptions about
the population are needed in order for the test to be valid.`,
	{ marks: 1 }
);
