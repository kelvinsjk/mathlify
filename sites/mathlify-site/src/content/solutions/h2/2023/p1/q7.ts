import { Answer } from '$content/solutions/answerObject';
import { mathlify, mathlifyQED } from '$lib/mathlifier';

export const answer = new Answer();

// a
{
	// TODO: graphs
	const soln = mathlifyQED`Asymptotes: ${`y=2`}
and ${`x=3`}`;
	const ans = mathlify`Asymptotes: ${`y=2`}
and ${`x=3`}.`;
	answer.addPart(ans, soln);
}
