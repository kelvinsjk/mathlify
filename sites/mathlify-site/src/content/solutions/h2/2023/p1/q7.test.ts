import { expect, it } from 'vitest';
import { answer } from './q7.soln';

it('2023p1q7', () => {
	expect(answer).toMatchSnapshot();
});
