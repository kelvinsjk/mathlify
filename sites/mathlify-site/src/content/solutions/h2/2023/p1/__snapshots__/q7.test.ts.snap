// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`2023p1q7 1`] = `
Answer {
  "answer": {
    "parts": [
      {
        "body": "<p><a href="#soln-1">Sketch</a>.</p>
",
      },
      {
        "body": "<p><math><mrow><msub><mi>R</mi><mi>f</mi></msub><mo>=</mo></mrow><mrow><mrow><mo fence="true" form="prefix">[</mo><mn>0</mn><mo separator="true">,</mo><mi>∞</mi><mo fence="true" form="postfix">)</mo></mrow><mi>.</mi></mrow></math></p>
",
      },
      {
        "body": "<p><math><msup><mi>f</mi><mn>2</mn></msup></math>
does not exist as <math><mrow><msub><mi>R</mi><mi>f</mi></msub><mo>⊈</mo><msub><mi>D</mi><mi>f</mi></msub><mi>.</mi></mrow></math></p>
",
      },
      {
        "body": "<p>Greatest value of <math><mrow><mi>a</mi><mo>=</mo><mo form="prefix" stretchy="false">−</mo><mn>2</mn><mi>.</mi></mrow></math></p>
",
      },
      {
        "body": "<p><math><mrow><msup><mi>f</mi><mrow><mo lspace="0em" rspace="0em">−</mo><mn>1</mn></mrow></msup><mo form="prefix" stretchy="false">(</mo><mi>x</mi><mo form="postfix" stretchy="false">)</mo><mo>=</mo></mrow><mrow><mfrac><mrow><mn>3</mn><mi>x</mi><mo>+</mo><mn>4</mn></mrow><mrow><mi>x</mi><mo>−</mo><mn>2</mn></mrow></mfrac><mi>.</mi></mrow></math>
<br>
<math><mrow><msub><mi>D</mi><msup><mi>f</mi><mrow><mo lspace="0em" rspace="0em">−</mo><mn>1</mn></mrow></msup></msub><mo>=</mo></mrow><mrow><msub><mi>R</mi><mi>f</mi></msub><mo>=</mo></mrow><mrow><mrow><mo fence="true" form="prefix">[</mo><mn>0</mn><mo separator="true">,</mo><mn>2</mn><mo fence="true" form="postfix">)</mo></mrow><mi>.</mi></mrow></math></p>
",
      },
    ],
  },
  "solution": {
    "parts": [
      {
        "body": "<svg
	xmlns="http://www.w3.org/2000/svg"
	class="graph svelte-1vox4to"
	viewBox="-55 -48 110.00000000000001 88"
>
	<style>
		foreignObject {
			color: black;
			font-size: 7px;
			overflow: visible;
			white-space: nowrap;
			display: grid;
		}
		foreignObject math {
			flex-wrap: nowrap;
			transform: translate(-50%, -50%);
		}
	</style>
	<defs>
		<marker id="closed" markerWidth="5" markerHeight="5" refX="1" refY="1">
			<circle cx="1" cy="1" r="1" fill="black"></circle>
		</marker>
		<marker id="open" markerWidth="5" markerHeight="5" refX="1" refY="1">
			<circle
				cx="1"
				cy="1"
				r="0.75"
				fill="white"
				markerUnits="strokeWidth"
				style="stroke: black; stroke-width: 0.5"
			></circle>
		</marker>
		<marker id="arrow" markerWidth="5" markerHeight="5" refX="0" refY="1" orient="auto">
			<path d="M 0 0 L 2 1 L 0 2 z" fill="black"></path>
		</marker>
	</defs>
	<line
		marker-end="url(#arrow)"
		x1="-50"
		x2="50"
		y1="30"
		y2="30"
		style="stroke: black; stroke-width: 2"
	></line>
	<line
		marker-end="url(#arrow)"
		x1="0"
		x2="0"
		y1="40"
		y2="-40"
		style="stroke: black; stroke-width: 2"
	></line>
	<foreignObject width="1" height="1" class="svelte-1vox4to" x="50" y="35"
		><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>x</mi></math
		><!----></foreignObject
	><foreignObject width="1" height="1" class="svelte-1vox4to" x="-5" y="-42"
		><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>y</mi></math
		><!----></foreignObject
	><!----><!---->
	<line stroke-dasharray="4" x1="15" x2="15" y1="40" y2="-40" style="stroke: green"></line>
	<foreignObject class="graph-label svelte-1vox4to" width="1" height="1" x="15" y="-40"
		><div style="display: grid" xmlns="http://www.w3.org/1999/xhtml" class="svelte-1vox4to">
			<math xmlns="http://www.w3.org/1998/Math/MathML"
				><mrow><mi>x</mi><mo>=</mo></mrow
				><mrow><mn>3</mn></mrow></math
			><!---->
		</div></foreignObject
	><!----><!---->
	<line stroke-dasharray="4" x1="-50" x2="50" y1="20" y2="20" style="stroke: green"></line>
	<foreignObject class="graph-label svelte-1vox4to" width="1" height="1" x="-50" y="15"
		><math xmlns="http://www.w3.org/1998/Math/MathML"
			><mrow><mi>y</mi><mo>=</mo></mrow
			><mrow><mn>2</mn></mrow></math
		><!----></foreignObject
	><!----><!---->
	<path
		d="M-50.000,23.846 -48.955,23.909 -47.910,23.974 -46.864,24.041 -45.819,24.111 -44.774,24.182 -43.729,24.257 -42.684,24.334 -41.639,24.414 -40.593,24.497 -39.548,24.583 -38.503,24.673 -37.458,24.766 -36.413,24.863 -35.368,24.964 -34.322,25.069 -33.277,25.178 -32.232,25.293 -31.187,25.413 -30.142,25.538 -29.097,25.669 -28.051,25.807 -27.006,25.951 -25.961,26.103 -24.916,26.263 -23.871,26.432 -22.826,26.609 -21.780,26.797 -20.735,26.996 -19.690,27.207 -18.645,27.431 -17.600,27.669 -16.555,27.923 -15.509,28.194 -14.464,28.485 -13.419,28.797 -12.374,29.133 -11.329,29.495 -10.284,29.888 -9.2384,29.686 -8.1932,29.221 -7.1481,28.712 -6.1029,28.153 -5.0577,27.536 -4.0125,26.851 -2.9674,26.086 -1.9222,25.227 -0.87703,24.254 0.16814,23.144 1.2133,21.867 2.2585,20.379 3.3036,18.626 4.3488,16.528 5.3940,13.975 6.4392,10.797 7.4843,6.7362 8.5295,1.3632 9.5747,-6.0801 10.620,-17.075 11.665,-34.963 "
		style="stroke: blue; fill: none"
	></path>
	<!---->
	<path
		d="M19.545,-35.006 20.595,-24.681 21.645,-17.620 22.696,-12.486 23.746,-8.5855 24.796,-5.5210 25.846,-3.0499 26.896,-1.0151 27.946,0.68958 28.997,2.1385 30.047,3.3851 31.097,4.4691 32.147,5.4202 33.197,6.2617 34.247,7.0112 35.298,7.6833 36.348,8.2892 37.398,8.8383 38.448,9.3382 39.498,9.7952 40.548,10.215 41.599,10.601 42.649,10.958 43.699,11.289 44.749,11.596 45.799,11.883 46.849,12.151 47.900,12.401 48.950,12.636 50.000,12.857 "
		style="stroke: blue; fill: none"
	></path>
	<foreignObject
		class="graph-label svelte-1vox4to"
		width="1"
		height="1"
		x="19.544999999999987"
		y="-35.00550055005502"
		><math xmlns="http://www.w3.org/1998/Math/MathML"
			><mrow><mi>y</mi><mo>=</mo></mrow
			><mrow
				><mrow
					><mo fence="true" form="prefix">|</mo
					><mfrac
						><mrow><mn>2</mn><mi>x</mi><mo>+</mo><mn>4</mn></mrow
						><mrow><mn>3</mn><mo>−</mo><mi>x</mi></mrow></mfrac
					><mo fence="true" form="postfix">|</mo></mrow
				></mrow
			></math
		><!----></foreignObject
	><!----><!---->
	<line style="stroke: black" y1="27" y2="33" x1="-10" x2="-10"></line>
	<foreignObject width="1" height="1" class="svelte-1vox4to" y="36" x="-10"
		><div style="display: grid" xmlns="http://www.w3.org/1999/xhtml" class="svelte-1vox4to">
			<math xmlns="http://www.w3.org/1998/Math/MathML"
				><mrow
					><mo form="prefix" stretchy="false">(</mo><mo form="prefix" stretchy="false">−</mo
					><mn>2</mn><mo separator="true">,</mo><mn>0</mn
					><mo form="postfix" stretchy="false">)</mo></mrow
				></math
			><!---->
		</div></foreignObject
	><!---->
	<line style="stroke: black" x1="-3" x2="3" y1="23.333333333333336" y2="23.333333333333336"></line>
	<foreignObject width="1" height="1" class="svelte-1vox4to" x="-10" y="23.333333333333336"
		><div
			class="label-left svelte-1vox4to"
			style="display: grid"
			xmlns="http://www.w3.org/1999/xhtml"
		>
			<math xmlns="http://www.w3.org/1998/Math/MathML"
				><mrow
					><mo fence="true" form="prefix">(</mo><mn>0</mn><mo separator="true">,</mo
					><mfrac><mn>4</mn><mn>3</mn></mfrac
					><mo fence="true" form="postfix">)</mo></mrow
				></math
			><!---->
		</div></foreignObject
	><!---->
</svg>
",
      },
      {
        "body": "<p><math><mrow><msub><mi>R</mi><mi>f</mi></msub><mo>=</mo></mrow><mrow><mrow><mo fence="true" form="prefix">[</mo><mn>0</mn><mo separator="true">,</mo><mi>∞</mi><mo fence="true" form="postfix">)</mo></mrow><mspace width="0.2778em"></mspace><mi>■</mi></mrow></math></p>
",
      },
      {
        "body": "<p><math display="block" class="tml-display" style="display:block math;"><mtable displaystyle="true" columnalign="right left"><mtr><mtd class="tml-right" style="padding:0.7ex 0em 0.7ex 0em;"><msub><mi>R</mi><mi>f</mi></msub></mtd><mtd class="tml-left" style="padding:0.7ex 0em 0.7ex 0em;"><mrow><mo>=</mo><mrow><mo fence="true" form="prefix">[</mo><mn>0</mn><mo separator="true">,</mo><mi>∞</mi><mo fence="true" form="postfix">)</mo></mrow></mrow></mtd></mtr><mtr><mtd class="tml-right" style="padding:0.7ex 0em 0.7ex 0em;"><msub><mi>D</mi><mi>f</mi></msub></mtd><mtd class="tml-left" style="padding:0.7ex 0em 0.7ex 0em;"><mrow><mo>=</mo><mrow><mo fence="true" form="prefix">(</mo><mo form="prefix" stretchy="false">−</mo><mi>∞</mi><mo separator="true">,</mo><mn>3</mn><mo fence="true" form="postfix">)</mo></mrow><mo>∪</mo><mrow><mo fence="true" form="prefix">(</mo><mn>3</mn><mo separator="true">,</mo><mi>∞</mi><mo fence="true" form="postfix">)</mo></mrow></mrow></mtd></mtr></mtable></math></p>
<p>Since <math><mrow><msub><mi>R</mi><mi>f</mi></msub><mo>⊈</mo></mrow><mrow><msub><mi>D</mi><mi>f</mi></msub><mo separator="true">,</mo></mrow></math>
the composite function <math><msup><mi>f</mi><mn>2</mn></msup></math>
does not exist <math><mrow><mspace width="0.2778em"></mspace><mi>∎</mi></mrow></math></p>
",
      },
      {
        "body": "<p>Greatest value of <math><mrow><mrow><mi>a</mi><mo>=</mo><mo form="prefix" stretchy="false">−</mo><mn>2</mn></mrow><mspace width="0.2778em"></mspace><mi>■</mi></mrow></math></p>
",
      },
      {
        "body": "<p><math display="block" class="tml-display" style="display:block math;"><mrow><mtext>Let </mtext><mi>y</mi><mo>=</mo><mrow><mo fence="true" form="prefix">|</mo><mfrac><mrow><mn>2</mn><mi>x</mi><mo>+</mo><mn>4</mn></mrow><mrow><mn>3</mn><mo>−</mo><mi>x</mi></mrow></mfrac><mo fence="true" form="postfix">|</mo></mrow></mrow></math></p>
<p>Since <math><mrow><mfrac><mrow><mn>2</mn><mi>x</mi><mo>+</mo><mn>4</mn></mrow><mrow><mn>3</mn><mo>−</mo><mi>x</mi></mrow></mfrac><mo>&lt;</mo><mn>0</mn></mrow></math>
for
<math><mrow><mi>x</mi><mo>≤</mo><mo form="prefix" stretchy="false">−</mo><mn>2</mn><mo separator="true" lspace="0em" rspace="0em">,</mo></mrow></math></p>
<p><math display="block" class="tml-display" style="display:block math;"><mtable displaystyle="true"><mtr><mtd style="padding:0.7ex 0em 0.7ex 0em;"><mrow><mi>y</mi><mo>=</mo><mo form="prefix" stretchy="false">−</mo><mfrac><mrow><mn>2</mn><mi>x</mi><mo>+</mo><mn>4</mn></mrow><mrow><mn>3</mn><mo>−</mo><mi>x</mi></mrow></mfrac></mrow></mtd></mtr><mtr><mtd style="padding:0.7ex 0em 0.7ex 0em;"><mrow><mi>y</mi><mrow><mo fence="true" form="prefix">(</mo><mn>3</mn><mo>−</mo><mi>x</mi><mo fence="true" form="postfix">)</mo></mrow><mo>=</mo><mo form="prefix" stretchy="false">−</mo><mn>2</mn><mi>x</mi><mo>−</mo><mn>4</mn></mrow></mtd></mtr><mtr><mtd style="padding:0.7ex 0em 0.7ex 0em;"><mrow><mn>3</mn><mi>y</mi><mo>−</mo><mi>y</mi><mi>x</mi><mo>=</mo><mo form="prefix" stretchy="false">−</mo><mn>2</mn><mi>x</mi><mo>−</mo><mn>4</mn></mrow></mtd></mtr><mtr><mtd style="padding:0.7ex 0em 0.7ex 0em;"><mrow><mi>y</mi><mi>x</mi><mo>−</mo><mn>2</mn><mi>x</mi><mo>=</mo><mn>3</mn><mi>y</mi><mo>+</mo><mn>4</mn></mrow></mtd></mtr><mtr><mtd style="padding:0.7ex 0em 0.7ex 0em;"><mrow><mi>x</mi><mrow><mo fence="true" form="prefix">(</mo><mi>y</mi><mo>−</mo><mn>2</mn><mo fence="true" form="postfix">)</mo></mrow><mo>=</mo><mn>3</mn><mi>y</mi><mo>+</mo><mn>4</mn></mrow></mtd></mtr><mtr><mtd style="padding:0.7ex 0em 0.7ex 0em;"><mrow><mi>x</mi><mo>=</mo><mfrac><mrow><mn>3</mn><mi>y</mi><mo>+</mo><mn>4</mn></mrow><mrow><mi>y</mi><mo>−</mo><mn>2</mn></mrow></mfrac></mrow></mtd></mtr></mtable></math></p>
<p><math display="block" class="tml-display" style="display:block math;"><mrow><msup><mi>f</mi><mrow><mo lspace="0em" rspace="0em">−</mo><mn>1</mn></mrow></msup><mo form="prefix" stretchy="false">(</mo><mi>x</mi><mo form="postfix" stretchy="false">)</mo><mo>=</mo><mfrac><mrow><mn>3</mn><mi>x</mi><mo>+</mo><mn>4</mn></mrow><mrow><mi>x</mi><mo>−</mo><mn>2</mn></mrow></mfrac><mspace width="0.2778em"></mspace><mi>■</mi></mrow></math></p>
<p><math display="block" class="tml-display" style="display:block math;"><mtable displaystyle="true" columnalign="right left"><mtr><mtd class="tml-right" style="padding:0.7ex 0em 0.7ex 0em;"><msub><mi>D</mi><msup><mi>f</mi><mrow><mo lspace="0em" rspace="0em">−</mo><mn>1</mn></mrow></msup></msub></mtd><mtd class="tml-left" style="padding:0.7ex 0em 0.7ex 0em;"><mrow><mo>=</mo><msub><mi>R</mi><mi>f</mi></msub></mrow></mtd></mtr><mtr><mtd class="tml-right" style="padding:0.7ex 0em 0.7ex 0em;"><mrow></mrow></mtd><mtd class="tml-left" style="padding:0.7ex 0em 0.7ex 0em;"><mrow><mo>=</mo><mrow><mo fence="true" form="prefix">[</mo><mn>0</mn><mo separator="true">,</mo><mn>2</mn><mo fence="true" form="postfix">)</mo></mrow><mspace width="0.2778em"></mspace><mi>■</mi></mrow></mtd></mtr></mtable></math></p>
",
      },
    ],
  },
}
`;
